pkg/compactv2/modifiers.go

```diff
@@ -0,0 +1,334 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package compactv2
+
+import (
+	"github.com/pkg/errors"
+	"github.com/prometheus/prometheus/pkg/relabel"
+	"github.com/prometheus/prometheus/storage"
+	"github.com/prometheus/prometheus/tsdb"
+	"github.com/prometheus/prometheus/tsdb/chunkenc"
+	"github.com/prometheus/prometheus/tsdb/chunks"
+	"github.com/prometheus/prometheus/tsdb/index"
+	"github.com/prometheus/prometheus/tsdb/tombstones"
+	"github.com/thanos-io/thanos/pkg/block/metadata"
+)
+
+type Modifier interface {
+	Modify(sym index.StringIter, set storage.ChunkSeriesSet, log ChangeLogger, p ProgressLogger) (index.StringIter, storage.ChunkSeriesSet)
+}
+
+type DeletionModifier struct {
+	deletions []metadata.DeletionRequest
+}
+
+var (
+	SupportedRelabelActions = map[relabel.Action]struct{}{relabel.Keep: {}, relabel.Drop: {}}
```

Comment:
This is not used

Reply:
It's used in many places :thinking:

Reply:
![image](https://user-images.githubusercontent.com/6950331/98965309-cda0e200-2501-11eb-86a0-c3425ae99aa9.png)

Reply:
That's another one named `SelectorSupportedRelabelActions`. This one is `SupportedRelabelActions`, I see they have different supported actions.

Reply:
ah got it. Removing now.